website/content/api-docs/secret/pki.mdx

```diff
@@ -2144,8 +2144,19 @@ This endpoint allows an operator to revoke an issuer certificate, marking it
 unable to issue new certificates and adding it to other issuers' CRLs, if they
 have signed this issuer's certificate. This will cause all CRLs to be rebuilt.
 
+This is mostly provided for book-keeping and as a soft-delete feature, to
+ensure this issuer is not accidentally reused in the future.
+
 ~> **Warning**: This operation cannot be undone!
 
+~> Note: This operation **does not** have any impact on other clusters or
+   mounts and **may not** have any impact on whether clients consider these
+   issuers revoked. Revoked issuers will not appear on their own CRLs. Revoked
+   issuers may not appear on other CRLs if a suitable parent is not present in
+   this mount point. Revoked issuers will still need to be revoked in any
```

Comment:
"CRLs if a suitable parent is not present in this mount point" I think would read better as "CRLs if a suitable parent is not present in the same mount point"?